package.json

```diff
@@ -33,7 +33,7 @@
     "yargs": "^12.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "export REACT_APP_LOGKEEPER_BASE=https://logkeeper.mongodb.org; export REACT_APP_EVERGREEN_BASE=https://evergreen.mongodb.com; react-scripts start",
```

Comment:
Are these variables needed in the build step?

Reply:
they are exported in evergreen https://github.com/evergreen-ci/evergreen/blob/main/scripts/update-lobster.sh#L29-L30